### ISSUE TITLE
Learning rate for fine tuning the model

### ISSUE THREAD

**Issue by bchinnari** [OHF-Voice/piper1-gpl#20]
Hi.

I am finetuning an "en" model with just one speaker data and that too i have just 10-15mins of it. It is already doing a good job in cloning.
But i have quick question. COuld you suggest the best parameters for fine-tuning with such small amount of data for just one speaker.

ANy suggestions for Paramaters like batch_size or learning rate.
